core/loadpoint.go

```diff
@@ -794,7 +792,7 @@ func (lp *Loadpoint) plannerActive() (active bool) {
 
 	if active {
 		// ignore short plans if not already active
-		if !lp.planActive && requiredDuration < 10*time.Minute {
+		if !lp.planActive && lp.clock.Now().Add(10*time.Minute).After(slotEnd) {
```

Comment:
lp.clock.Until() < xx

Reply:
10min als const:

    smallSlotDuration = 10 * time.Minute // small planner slot duration we might ignore